pages/common/cf.md

```diff
@@ -0,0 +1,32 @@
+# cf
+
+> Command line tool to manage apps and services on Cloud Foundry.
+> More information: <https://docs.cloudfoundry.org/>.
+
+- Push an app using the default settings:
+
+`cf push {{app_name}}`
+
+- View the services available from your organization:
+
+`cf marketplace`
+
+- Create a service instance:
+
+`cf create-service {{service}} {{plan}} {{chosen_service_name}}`
+
+- Connect an application to a service:
+
+`cf bind-service {{app_name}} {{chosen_service_name}}`
```

Comment:
The service is named by the user, `service_name` is not enough clear about that?
```suggestion
`cf bind-service {{app_name}} {{service_name}}`
```